### PR TITLE
[native]Add TableWriterMergeNode in presto protocol

### PR DESCRIPTION
**Pull request by xiaoxmeng**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h [hunk 1]
```diff
@@ -50,36 +50,39 @@ extern const char* const PRESTO_BUFFER_COMPLETE_HEADER;
...
 extern const char* const PRESTO_MAX_SIZE_DEFAULT;
 
-
 extern const char* const PRESTO_ABORT_TASK_URL_PARAM;
 
 class Exception : public std::runtime_error {
  public:
-  explicit Exception(const std::string& message) : std::runtime_error(message) {};
+  explicit Exception(const std::string& message)
+      : std::runtime_error(message){};
```
Comment (yingsu00) [inline presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h:58]: There are so many formatting changes and it's difficult to see what's actually changed. Would it be possible to exclude the pure formatting changes?

Comment (mbasmanova) [reply presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h:58]: @yingsu00 These files are auto-generated by running `make protocol`.

**Review by mbasmanova — APPROVED** [review]
@xiaoxmeng Thanks.

**Comment by mbasmanova** [comment]
@xiaoxmeng There are CI failures:

```
[ERROR] Failures: 
[ERROR]   TestPrestoNativeAggregations>AbstractTestQueryFramework.init:84->createExpectedQueryRunner:31 ? Creation Unable to create injector, see the following errors:

1) Configuration property 'hive.allow-drop-table' was not used
  at com.facebook.airlift.bootstrap.Bootstrap.lambda$initialize$2(Bootstrap.java:244)

1 error
[ERROR]   TestPrestoNativeJoinQueries>AbstractTestQueryFramework.init:84->createExpectedQueryRunner:31 ? Creation Unable to create injector, see the following errors:

1) Configuration property 'hive.allow-drop-table' was not used
  at com.facebook.airlift.bootstrap.Bootstrap.lambda$initia
```

**Comment by xiaoxmeng** [comment]
> 

@mbasmanova thanks for the quick review. Let me rebase and the hive drop table option change has been reverted.

**Comment by amitkdutta** [comment]
> > 
> 
> @mbasmanova thanks for the quick review. Let me rebase and the hive drop table option change has been reverted.



> @xiaoxmeng There are CI failures:
> 
> ```
> [ERROR] Failures: 
> [ERROR]   TestPrestoNativeAggregations>AbstractTestQueryFramework.init:84->createExpectedQueryRunner:31 ? Creation Unable to create injector, see the following errors:
> 
> 1) Configuration property 'hive.allow-drop-table' was not used
>   at com.facebook.airlift.bootstrap.Bootstrap.lambda$initialize$2(Bootstrap.java:244)
> 
> 1 error
> [ERROR]   TestPrestoNativeJoinQueries>AbstractTestQueryFramework.init:84->createExpectedQueryRunner:31 ? Creation Unable to create injector, see the following errors:
> 
> 1) Configuration property 'hive.allow-drop-table' was not used
>   at com.facebook.airlift.bootstrap.Bootstrap.lambda$initia
> ```

Its from a local master. The allow-drop-table option is reverted https://github.com/prestodb/presto/pull/20025

**Pull request merged**